app/views/comments/_feed.html.erb

```diff
@@ -1,16 +1,19 @@
-<h3>Comments <span class="badge" id="comment-count"><%= commentable.comments.count %></span></h3>
-<div class="comment-feed">
-  <div class="comment-list">
-    <%= render(commentable.comments.includes(:user)) if commentable.comments.any? %>
+<% current_user_comments = commentable.comments.where(user: current_user) %>
+<div class="comment-feed" data-behavior="comment-feed" data-current-user-comments="<%= current_user_comments.map(&:id)%>">
+<% cache ['comment-feed', commentable.comments] do %>
+  <h3>Comments <span class="badge" id="comment-count"><%= commentable.comments.count %></span></h3>
+    <div class="comment-list">
+      <%= render(commentable.comments.includes(:user)) if commentable.comments.any? %>
 
-    <div data-notice="no-comments" class="comment"
-      <% if commentable.comments.any? %>
-        style="display: none;" <%# may be un-hidden later by JS %>
-      <% end %>
-    >
-      There have been no comments yet.
+      <div data-notice="no-comments" class="comment"
+        <% if commentable.comments.any? %>
+          style="display: none;" <%# may be un-hidden later by JS %>
```

Comment:
Can we use the `hidden` class here instead of adding a `style` attribute?

Reply:
👍